### ISSUE TITLE
style service related objects

### ISSUE THREAD

**Issue by HMT-HRO-MaMe** [indoorequal/indoorequal.org#52]
Hi, the style is pretty nice and modern. But I noticed that currently it's not easy to recognize where a guest of the building can get things, services or support.

How about a different style for rooms like restaurant, toilets, amphitheatre, lecturer? So they are easy to recognize and it's a tip easier to know how to get there to get what you are looking for (within the building).

**Comment by HMT-HRO-MaMe** [comment]
I guess hall would be another interesting value 😉

**Comment by francois2metz** [comment]
For restaurant and toilets, I would suggest to use standard POI tags such as `amenity=toilets` or `amenity=restaurant`. It will be displayed fine.

For others rooms tagged with room=_servicesroom_, we could style them the same color as POI. Do you have a list of rooms values that could make sense?

**Comment by HMT-HRO-MaMe** [comment]
Yes true, I tagged the POIs on the indoor=room outlines and it already get's some more structures:
![Screenshot 2022-08-08 at 08-58-26 indoor](https://user-images.githubusercontent.com/102375018/183386981-8142f939-2648-4674-91e9-8c8c51687723.png)


When I browse trough taginfo, I tend to suggest the following which touch aspects of very different buildings like conference halls, airports, shopping malls, ...

Events
- conference
- amphitheatre  (not sure, this is suggested by osminedit?)
- hall
- lobby
- foyer
- lounge
- terrace


Tasks
- class
- auditorium
- lecture_hall
- library
- seminar | seminar_room
- laboratory
- computer
- studio
- washroom
- fitness
- sports
- gym
- break
- kitchen

Shopping
- store
- retail
- commercial
- cafe
- cafeteria
- restaurant

**Comment by francois2metz** [comment]
I tried to style some room values on a specific branch of the plugin: https://github.com/indoorequal/mapbox-gl-indoorequal/tree/test-subclass with values "class", "laboratory", "office", "auditorium", "amphitheatre", "reception".
Here is some results on your university:

![Screenshot 2022-08-08 at 22-45-48 Map with indoor](https://user-images.githubusercontent.com/86659/183512064-18461bcb-5f37-4165-b3f1-aa8564ef368f.png)
![Screenshot 2022-08-08 at 22-46-04 Map with indoor](https://user-images.githubusercontent.com/86659/183512117-3323f38a-e86d-4ed6-9b0f-a5f8f7dcff07.png)

**Comment by HMT-HRO-MaMe** [comment]
Looks very promising to me :-)

**Comment by HMT-HRO-MaMe** [comment]
So what do you think? Can you push it live? _(I would like to get some feedback by other university departments if this would be a good way to show rooms)_

P.S: I noticed in the screenshot, that huge halls that are mapped room=amphitheatre seem to be missing?

**Comment by francois2metz** [comment]
> So what do you think? Can you push it live? _(I would like to get some feedback by other university departments if this would be a good way to show rooms)_

I'm waiting for the new import first.

> P.S: I noticed in the screenshot, that huge halls that are mapped room=amphitheatre seem to be missing?

It seems that it's tagged as `indoor=yes`. You should tag it as a room or similar.

**Comment by HMT-HRO-MaMe** [comment]
Oh, thanks, what a stupid mistake 😞

**Comment by francois2metz** [comment]
Hi @HMT-HRO-MaMe, I published the experimental style. It's live!

**Comment by HMT-HRO-MaMe** [comment]
Thank you very much, this commes with really great details for our campus 👍
